content/zh/docs/reference/kubernetes-api/authorization-resources/subject-access-review-v1.md

```diff
@@ -0,0 +1,384 @@
+---
+api_metadata:
+  apiVersion: "authorization.k8s.io/v1"
+  import: "k8s.io/api/authorization/v1"
+  kind: "SubjectAccessReview"
+content_type: "api_reference"
+description: "SubjectAccessReview 检查用户或组是否可以执行操作。"
+title: "SubjectAccessReview"
+weight: 4
+auto_generated: true
```

Comment:
nit: the localized page is _not_ auto-generated.

Reply:
Although, technically, I should have made sure a CLA was in place before reviewing. Please see https://github.com/kubernetes/website/pull/33523#issuecomment-1119526920 @shixiuguo

Reply:
Yes, line 10 should be removed.